src/util/path.ts

```diff
@@ -99,13 +103,19 @@ export function resolvePath(normalizedPath: string, origin: string): string {
     return normalizedPath;
   }
   return normalizePath(
-    dirname(origin) + String.fromCharCode(separator) + normalizedPath
+    dirname(origin) + PATH_DELIMITER + normalizedPath
```

Comment:
Looks like this is ignoring separator now

Reply:
What do mean? I should revert this?

Reply:
It's now a mix between using `separator` in other places and `PATH_DELIMITER` here. Initial idea of the path utility was that it should theoretically also work with windows paths when switching out `separator`, and it shouldn't use `PATH_DELIMITER` which is always `/` I think.

Reply:
Wait, normalize won't ever produce windows paths most likely, so it might be ok.